src/System.Security.Cryptography.Cng/ref/System.Security.Cryptography.Cng.csproj

```diff
@@ -5,11 +5,14 @@
     <DefineConstants Condition="'$(TargetGroup)' == 'netfx'">$(DefineConstants);netfx</DefineConstants>
     <NoWarn Condition="'$(TargetGroup)' == 'netfx'">$(NoWarn);0436</NoWarn>
     <ProjectGuid>{9FD12550-3A7C-49D3-9A1E-C4B7410989DD}</ProjectGuid>
+    <DefineConstants Condition="'$(TargetGroup)' == 'uap' Or '$(TargetGroup)' == 'netcoreapp'">$(DefineConstants);FEATURE_HASHDATA</DefineConstants>
```

Comment:
While I'm generally more fond of HAVE_ and FEATURE_ type things, I thought we had "standardized" on the runtime identifiers for our \#defines.

@weshaggard?